src/WorldStorage/WSSAnvil.h

```diff
@@ -92,6 +92,8 @@ class cWSSAnvil :
 		bool OpenFile(bool a_IsForReading);
 	} ;
 	typedef std::list<cMCAFile *> cMCAFiles;
+	typedef void (cWSSAnvil::*EntityLoaderFunc)(cEntityList &, const cParsedNBT &, int a_EntityTagIdx);
+	typedef std::map<AString, EntityLoaderFunc> EntityLoaderMap;
```

Comment:
These could be declared in the cpp file - they are not part of the loader's public API anyway.